.github/workflows/lint-test-build.yml

```diff
@@ -0,0 +1,55 @@
+name: Lint / Test / Build
+
+on:
+  pull_request:
+
+
+env:
+  DEFAULT_NODE_VERSION: 18.x
+
+permissions:
+  contents: read
+
+jobs:
+
+  build:
+    name: ${{ matrix.action }} (${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [18.x, 20.x]
```

Comment:
Is there a difference between `18.x` and `18` for this and `DEFAULT_NODE_VERSION`?

Reply:
Thank you. I was just wondering if using `18.x` would produce a different result to `18`. However with the recent update it seems like it doesn't.